src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html

```diff
@@ -169,3 +194,38 @@
     <ng-template ngPluralCase="other">{{ title }}s</ng-template>
   </span>
 </ng-template>
+
+<ng-template #dropdownTemplate>
+  <ng-container *ngFor="let data of dropdownData.total.category | keyvalue">
+    <li class="list-group-item">
+      <div class="d-flex pl-1 pb-2 pt-2">
+        <div class="ms-5 me-auto">
+          <span *ngIf="data.value.total"
+                [ngPlural]="data.value.total">
+              {{ data.value.total }}
+            <ng-template ngPluralCase="=0">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="=1">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="other">{{ hwNames[data.key] }}s</ng-template>
```

Comment:
?
```suggestion
            <ng-template ngPluralCase="other">{{ hwNames[data.key] | pluralize }}s</ng-template>
```

where pluralize can be a pipe that has something like
```
if (value.endsWith('y')) {
      return value.slice(0, -1) + 'ies';
    } else {
      return value + 's';
    }
 ```

BTW, I haven't tested the above so you'll have to do that.

Reply:
Done